netbox/release.yaml

```diff
@@ -0,0 +1,2 @@
+version: "4.1.0"
```

Comment:
This is fine, but wondering if we should just use pyproject.toml as it should already have the version info in it - the designation would just be from the version string - I think there are standard libraries to parse the versioning.

Reply:
yes, but not set on it - whatever is easiest for Nat to work with.

Reply:
AFAICT `importlib` can only glean metadata from a project that's actually been _installed_ as a package; it can't be used for local development. I feel like this is leading to a much deeper discussion around packaging and distribution, which I'm happy to have but I'd prefer not to block this particular FR.

Reply:
okay, just keep it in the yaml file then.